generators/tracegen/tracegen.mk

```diff
@@ -0,0 +1,18 @@
+##############################################################
+# extra variables/targets ingested by the chipyard make system
+##############################################################
+
+AXE_DIR=$(base_dir)/tools/axe/src
+AXE=$(AXE_DIR)/axe
+
+$(AXE): $(wildcard $(AXE_DIR)/*.[ch]) $(AXE_DIR)/make.sh
+	cd $(AXE_DIR) && ./make.sh
+
+$(output_dir)/tracegen.out: $(sim)
+	mkdir -p $(output_dir) && $(sim) $(PERMISSIVE_ON) +dramsim +max-cycles=$(timeout_cycles) $(SIM_FLAGS) $(EXTRA_SIM_FLAGS) $(VERBOSE_FLAGS) $(PERMISSIVE_OFF) none </dev/null 2> $@
```

Comment:
Why does this still use +dramsim and +max-cycles?

Reply:
I missed this. I only find/replaced in the `common.mk` and `Makefiles`